package.json

```diff
@@ -115,10 +121,18 @@
   "lint-staged": {
     "*.css": "stylelint",
     "*.{js,cjs,mjs}": "eslint --max-warnings=0",
-    "*.ts": "eslint --max-warnings=0 --plugin tsc --rule 'tsc/config: [2, {configFile: \"./tsconfig.json\"}]'",
+    "**/!(raidboss/data|oopsyraidsy/data)/*.ts": [
+      "dprint check"
+    ],
```

Comment:
Is this the only way to handle multiple dprint config files?

Does that mean if I run `dprint check` from the command-line, it'll reformat trigger files incorrectly?

Reply:
No it won't. actually `dprint check` will ignore trigger files. and just like `dprint fmt`.

dprint doesn't support config override like prettier https://prettier.io/docs/en/configuration.html#configuration-overrides

Reply:
maybe adding a `format` script